src/components/MaterialUI/Inputs/DatePicker/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import styled, { createGlobalStyle } from "styled-components";
+import { getThemeProp } from "../../../../utils";
+import Icon from "../../../../components/Icon";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import TimePicker from "./TimePicker";
+
+const CalendarIcon = styled(Icon).attrs(props => ({
+	id: getThemeProp(["icons", "date"], "calendar")(props),
+}))`
+	font-size: 20px;
+	padding: 3px 3px 3px 3px;
+`;
+
+const DatePickerWrapper = styled.label`
+	display: flex;
+	flex-wrap: wrap;
+	width: auto;
+	padding-left: 5px;
+`;
+
+const DatePickerStyle = createGlobalStyle`
+    .react-datepicker {
+        font-family: ${getThemeProp(["fonts", "base"], "sans-serif")};
+    }
+
+    .react-datepicker-popper {
+        z-index: 100;
+    }
+
+    .react-datepicker__input-container input {
+        width: 135px;
+        border: 1px solid ${getThemeProp(["colors", "borderLight"], "#cccccc")};
+        z-index: 100;
+    }
+`;
+
+const wrappedDatePicker = ({ value, useTime, onChange, dateFormat, showTimeZone, timeInputLabel, ...props }) => {
+	const parsedValue = new Date(value || "1970/01/01");
+
+	// eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Why was this disabled, you would need a pretty good reason to get rid of a "rule-of-hooks" warning ?

Reply:
It was due to not using an uppercase letter...  removed disable.